src/app/modules/item/pages/item-display/item-display.component.html

```diff
@@ -6,9 +6,31 @@
       </a>
     </nav>
   </ng-container>
-  <alg-error *ngIf="state.isError" i18n-message message="Unable to load the task"></alg-error>
+  <div *ngIf="state.isError" class="errors">
+    <alg-error i18n-message message="Unable to load the task"></alg-error>
+    <ng-container *ngIf="['all', 'all_with_grant'].includes(canEdit)">
+      <alg-error
+        *ngIf="initError$ | async; else urlError"
+        i18n-message message="It usually occurs when task url is invalid, if the task url is valid and the problem persists, please contact us."
+      ></alg-error>
+
+      <ng-template #urlError>
+        <alg-error
+          *ngIf="urlError$ | async as error; else unknownError"
+          message="{{ error.message }}"
+        ></alg-error>
+      </ng-template>
```

Comment:
Now you display the error... ok for the protocol stuff. But is there one place which verifies that the url is "valid" (so "*$!!" or "ftp://example.com" would not)? If yes, what is the message in this case? Is it localized?

Reply:
The errors are only displayed to task editors, and as I understood we intend to display a message when the task editor has the ability to solve the problem.
Indeed, I should check url validity in general **and** the protocol. I'll use the `URL` constructor since it fails when uri is invalid.